addons/io/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitSettings.java

```diff
@@ -0,0 +1,190 @@
+package org.openhab.io.homekit.internal;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.Dictionary;
+
+import org.osgi.framework.FrameworkUtil;
+
+/**
+ * Provides the configured and static settings for the Homekit addon
+ *
+ * @author Andy Lintner
+ */
+public class HomekitSettings {
+
+    private final static String NAME = "OpenHAB Homekit Bridge";
```

Comment:
Is this the name that is displayed when finding the bridge on iOS? Maybe it should simply be called "openHAB"?